Web/Assignment-1/OnLoadFunction.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE>
+<html>
+<head>
+<meta http-equiv="Content-type" content="text/html;charset=UTF-8">
+<script src="OnLoadJquery.js" type="text/javascript"></script>
```

Comment:
Either write jquery in js file or write in html page. You have write same js code in Js file and Html page. Remove one of them.